ports/descent2/Descent 2.sh

```diff
@@ -13,29 +13,35 @@ else
 fi
 
 source $controlfolder/control.txt
-source $controlfolder/device_info.txt
 [ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 get_controls
 
+# Variables
 GAMEDIR="/$directory/ports/descent2"
 DEVICE_ARCH="${DEVICE_ARCH:-aarch64}"
 GAME="d2x-rebirth"
 ASPECT_X=${ASPECT_X:-4}
 ASPECT_Y=${ASPECT_Y:-3}
 
+# CD and set permissions
 cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+rm -rf "$GAMEDIR/config/gamelog.txt"
+$ESUDO chmod +x -R $GAMEDIR/*
```

Comment:
We need execution flag on every files and folders  ?

Reply:
> It needs the execution flag on every files and folders ?

PortMaster does this anyway on install, this line exists for those who manually install it from the website.